### DISCUSSION TITLE
How do I set title for each tab in a tabset of a dashboard which contains iframe elements?

### DISCUSSION THREAD

**Discussion by lequangphu** [quarto-dev/quarto-cli#8013] [Q&A] [answered]
### Description

I have a tabset with 3 tabs like below, how do I customize title for each tab instead of showing tab 1, 2, 3?

![Screenshot 2023-12-22 at 16 49 22](https://github.com/quarto-dev/quarto-cli/assets/33654804/9c92a2b1-088e-4332-84ab-45e4670062ac)

````qmd
---
title: "My Dashboard"
format: dashboard
---

## row

### tabset1 {.tabset title="Tabset 1"}

<iframe></iframe>
<iframe></iframe>
<iframe></iframe>

````

I tried wrapping `div` around each iframe like below but the title was still `Tab 1`.

````qmd
::: {title="Iframe 1"}

<iframe></iframe>

:::
````

**Comment by mcanouil** [comment]
See <https://quarto.org/docs/interactive/layout.html#tabset-panel>.

For more, please share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

**Reply by lequangphu** [reply]
My question is about tabset in a dashboard https://quarto.org/docs/dashboards/layout.html#tabsets.

I just added my dashboard metadata to my question to make it reproducible.

**Reply by mcanouil** [reply, marked as answer]
Why don't you follow the examples in the documentation?
I don't expect title in random div to have any meaning.
The examples show: a card example, code cells example, and heading example.
Don't make up syntaxes.
In your case, the tabset title should be defined with headings as in regular document type.


Below nearly an exact copy of the example in <https://quarto.org/docs/dashboards/layout.html#tabsets> which should just work, if not, then it's a bug.

````

---
title: "My Dashboard"
format: dashboard
---

## row {.tabset}

### Tabset 1

<iframe></iframe>

### Tabset 2

<iframe></iframe>

### Tabset 3

<iframe></iframe>
````

**Reply by lequangphu** [reply]
Thanks! I should have read the document more thoroughly.
